pulumi/grapl/__main__.py

```diff
@@ -196,14 +196,19 @@ def main() -> None:
         "pipeline-ingress-healthcheck-polling-interval-ms",
         pipeline_ingress_healthcheck_polling_interval_ms,
     )
-    # This is a hack, because pipeline-ingress is not a kafka consumer, only a
-    # producer. The only consumers of this topic are integration tests. That is
-    # why we don't look up the consumer group name in the ccloud-bootstrap
-    # stack outputs (because it's absent).
-    pipeline_ingress_kafka_consumer_group_name = "pipeline-ingress-test"
+
+    pulumi.export(
+        "integration-tests-kafka-consumer-group-name",
+        kafka.consumer_group("integration-tests"),
+    )
+    integration_tests_kafka_credentials = kafka.service_credentials("integration-tests")
+    pulumi.export(
+        "integration-tests-kafka-sasl-username",
+        integration_tests_kafka_credentials.apply(lambda c: c.api_key),
+    )
     pulumi.export(
-        "pipeline-ingress-kafka-consumer-group-name",
-        pipeline_ingress_kafka_consumer_group_name,
+        "integration-tests-kafka-sasl-password",
+        integration_tests_kafka_credentials.apply(lambda c: c.api_secret),
```

Comment:
Purely hypothetically, couldn't these be declared in the `integration_tests/__main__.py`? That way you could skip the whole export-import song and dance, plus it's better namespacing.

Reply:
Not saying you gotta do this, but just raising it as a possibility.

Reply:
Yeah, I think they can. It'll take constructing the appropriate `Kafka()` but it seems doable. I'll give that a shot.

Reply:
fixed